### DISCUSSION TITLE
paru updating removed localrepo packages

### DISCUSSION THREAD

**Discussion by mqus** [Morganamilo/paru#381] [Q&A] [answered]
I have set up a local repo and let paru manage it. but whenever I uninstall a package that I installed via paru (e.g. by `sudo pacman -Rss`) paru will still try to update(and install) this package because it stayed in the local repo. I can run `repo-remove` by myself but I have to remember that and it is really cumbersome. 

Am I doing something wrong or is this (tracking actually installed packages) a missing feature?

**Comment by mqus** [comment]
Ah right, another thing that bothers me is that `paccache` also removes packages from that folder but also does not interface with the repo database, so the repo file (database) will say that a package still exists when it in fact does not. Can I solve this somehow?

**Comment by Morganamilo** [comment, marked as answer]
Yes if a package is in your local repo it's assumed you want it to be kept up to date. It shouldn't actually install the package though. At least it a chroot it doesn't.

With paccache, just tell it to only look at the primary cache dir. You can also just not make the repo a cachedir but then the packages will be copied into your cachedir to install.

**Reply by mqus** [reply]
Thank you!
So if I don't want paru to track a package I have to repo-remove it, right? or has paru a function for that?

**Reply by Morganamilo** [reply]
See `paru -L` in the man page.
